### PR TITLE
Fgts 83 add bird

### PR DESCRIPTION
**Pull request by jDenyes**
Added bird
Swoops at player
Damages player if player touches beak

### CODE REVIEW

File: godotproject/Level0.tscn [hunk 1]
```diff
@@ -128,9 +135,17 @@ rotation = 0.0108005
...
 [node name="AudioStreamPlayer" type="AudioStreamPlayer" parent="."]
 stream = ExtResource( 14 )
 autoplay = true
+=======
+position = Vector2( 9928, 544 )
+
+[node name="Bird" parent="." instance=ExtResource( 14 )]
+position = Vector2( 918.533, 493.948 )
+[connection signal="body_entered" from="DeathPlane" to="DeathPlane" method="_on_DeathPlane_body_entered"]
+>>>>>>> 0974653... Bird chnages direction
```
Comment (fheysel) [inline godotproject/Level0.tscn]: conflict needs to be resolved

File: godotproject/project.godot [hunk 2]
```diff
@@ -16,22 +16,13 @@ _global_script_class_icons={
 [application]
 
 config/name="Frog Goes To Space - Cooper Prototype"
-run/main_scene="res://Menus/MainMenu/MainMenu.tscn"
+run/main_scene="res://Level0.tscn"
```
Comment (fheysel) [inline godotproject/project.godot]: undo this

File: godotproject/Level0.tscn [hunk 3]
```diff
@@ -13,7 +13,14 @@
...
 [ext_resource path="res://Level0.gd" type="Script" id=12]
 [ext_resource path="res://EOL.tscn" type="PackedScene" id=13]
+<<<<<<< HEAD
 [ext_resource path="res://sfx/Level0-Theme.ogg" type="AudioStream" id=14]
+=======
+[ext_resource path="res://Bird.tscn" type="PackedScene" id=14]
+
+[sub_resource type="RectangleShape2D" id=1]
+extents = Vector2( 6585.7, 100 )
+>>>>>>> 0974653... Bird chnages direction
```
Comment (fheysel) [inline godotproject/Level0.tscn]: conflict

**Comment by fheysel** [comment]
still a bunch of merge conflicts and I cant get the branch to run in godot

File: godotproject/Bird.tscn [hunk 4]
```diff
@@ -0,0 +1,108 @@
+[gd_scene load_steps=15 format=2]
+
+[ext_resource path="res://birds.png" type="Texture" id=1]
+[ext_resource path="res://Bird.gd" type="Script" id=2]
+
```
Comment (charasyn) [inline godotproject/Bird.tscn:5]: It looks like bird.jpg is unused, can you remove?

**Review by charasyn — APPROVED** [review]

**Pull request merged**